.docker/parameters.yml

```diff
@@ -33,3 +33,18 @@ parameters:
         '16846': https://repro.elifesciences.org/example.html
     cache_control: private, no-cache, no-store, must-revalidate
     feature_xpub: true
+    calls_to_action:
+      - text: Here's something important
+        button:
+            text: Sign up for eLife news
+            path: https://elifesciences.org/
+        image:
+            alt: ''
+            uri: https://iiif.elifesciences.org/lax/09560%%2Felife-09560-fig1-v1.tif
```

Comment:
this is just a placeholder, but has a typo in the double `%`

Reply:
No, that's escaping as Symfony uses `%` to refer to parameters...

Reply:
Ok, will add it